docs/csharp/programming-guide/types/walkthrough-creating-and-using-dynamic-objects.md

```diff
@@ -174,4 +174,4 @@ The next project that you create in this walkthrough accesses a library that is
  [Using Type dynamic](../../../csharp/programming-guide/types/using-type-dynamic.md)  
  [Early and Late Binding](../../../visual-basic/programming-guide/language-features/early-late-binding/index.md)  
  [dynamic](../../../csharp/language-reference/keywords/dynamic.md)  
- [Implementing Dynamic Interfaces (external blog)](http://go.microsoft.com/fwlink/?LinkId=230895)
+ [Implementing Dynamic Interfaces](https://social.technet.microsoft.com/wiki/contents/articles/24446.mvp-articles-for-c-and-visual-basic.aspx#ImplementingDynamicInterfaces)
```

Comment:
I found this one confusing, since the link goes to a list of articles, and because the anchor is included in the link, the Implementing Dynamic Interfaces item is included at the top of the page, but it's still not the topic itself. I think that it might be best to include "(downloadable PDF from Microsoft TechNet)" as part of the link text, and include the PDF URL as the URL -- it's \[http://download.microsoft.com/download/5/4/B/54B83DFE-D7AA-4155-9687-B0CF58FF65D7/implementing-dynamic-interfaces.pdf].

Reply:
Indeed, it's better, I'll make that